vms/platformvm/service.go

```diff
@@ -1985,9 +1985,26 @@ func (service *Service) GetTxStatus(_ *http.Request, args *GetTxStatusArgs, resp
 	return nil
 }
 
+// StakedOut is a staked output
+type StakedOut struct {
+	// This output provides stake for this node ID
+	NodeID string `json:"nodeID"`
+	// Time at which this output will be returned to its owner(s).
+	// That is, the time at which the corresponding staker leaves the staker set.
+	StakedUntil json.Uint64 `json:"stakedUntil"`
+	// Time past which this output can be used for something other than staking.
+	StakeOnlyUntil json.Uint64 `json:"stakeOnlyUntil"`
+	// This output can be spent with [Threshold] signatures from [Owners]
+	Owners    []string    `json:"owners"`
+	Threshold json.Uint64 `json:"threshold"`
+	// Amount of this output
+	Amount json.Uint64 `json:"amount"`
+}
+
 // GetStakeReply is the response from calling GetStake.
 type GetStakeReply struct {
-	Staked json.Uint64 `json:"staked"`
+	Staked     json.Uint64 `json:"staked"`
+	StakedOuts []StakedOut `json:"stakedOuts"`
```

Comment:
I think we should return the same format that we return with `GetUTXOs`, it's going to be confusing if we implement things multiple different ways. Also the binary format is more general.